lib/device.toit

```diff
@@ -2,172 +2,87 @@
 // Use of this source code is governed by an MIT-style license that can be
 // found in the lib/LICENSE file.
 
-import device_impl_ as impl
 import uuid
 import encoding.tison
 
 /**
 Functionality available on devices (ESP32).
 */
 
-interface Device_:
-  name -> string?
-  hardware_id -> uuid.Uuid
-  estimate_time_accuracy -> int?
-
-
 /** Name of this device. */
-name -> string?:
-  return impl.Device_.instance.name
+name -> string:
+  return hardware_id.stringify
 
 /** Hardware ID of this device. */
-hardware_id/uuid.Uuid ::= impl.Device_.instance.hardware_id
-
-/**
-Estimated time accuracy of this device.
-
-On the device, the time drifts. The time is corrigated using the NTP when the
-  devices goes online. The estimated time accuracy is the estimated time
-  accuracy when the time was last set plus the an estimated time drift.
-*/
-estimate_time_accuracy -> int?:
-  return impl.Device_.instance.estimate_time_accuracy
+hardware_id/uuid.Uuid ::= uuid.uuid5 "hw_id" get_mac_address_
 
 /** Simple key-value store. */
 interface Store:
-  get name/string
-  delete name/string
-  set name/string value/any
+  get name/string -> any
+  delete name/string -> none
```

Comment:
We need to decide whether we want to call these "delete", or "remove".
I think we use the term "remove" in the core Map.